### PR TITLE
i3569 - Admissions pdf fix

### LINKED ISSUE #3569: Update pdf styles for admissions
- [x] Fix the css styles so it doesn't break before a table 
- [x] Update row span -> separated rows and updated thead vs tbody so consistent and distinguishes the table headers from the content associated with them. A majority of the tables had text above it for the title - so made all tables do that, and applied the header style to title instead of using a p tag to make it known that it's the title. Updated pdf scss to distinguish difference from header style vs paragraph

### PR DESCRIPTION
**Pull request by dnkemper**
Closes #3569 

# How to test
1. Checkout branch
2. blt ds --site=admissions.uiowa.edu
3. blt frontend
4. drush cr
5. login to admissions site locally
6. Look at pdfs and make sure tables break at a row - doesn't cut it off. Also make sure tables don't start at next page - and that they start right after content
https://admissions.local.drupal.uiowa.edu/print/pdf/node/451
https://admissions.local.drupal.uiowa.edu/print/pdf/node/446
https://admissions.local.drupal.uiowa.edu/print/pdf/node/456

### CODE REVIEW

**Comment by dnkemper** [comment]
Note: update module file to remove duplicate code

```
/**
 * Implements hook_preprocess_HOOK().
 */
function admissions_core_preprocess_entity_print(&$variables) {
  // Change the default PDF title from "View PDF" to the node title.
  if ($variables['theme_hook_original'] == 'entity_print__node__area_of_study'
    || $variables['theme_hook_original'] == 'entity_print__node__page') {
    $node = $variables['content'][0]['#node'] ?? $variables['content'][0][0]['#node'] ?? NULL;

    if ($node) {
      $variables['title'] = t('@title', [
        '@title' => $node->getTitle(),
      ]);
    }
  }
} 
```

File: docroot/sites/admissions.uiowa.edu/modules/admissions_core/sass/pdf.scss [hunk 1]
```diff
@@ -478,6 +473,15 @@ h6 {
   }
 }
 .node--type-page {
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    font-family: $font-family-serif;
```
Comment (bspeare) [inline docroot/sites/admissions.uiowa.edu/modules/admissions_core/sass/pdf.scss:481]: Do you mind removing the uppercase on these?  I don't think we have any styles for Zilla that use all caps.

File: docroot/sites/admissions.uiowa.edu/modules/admissions_core/sass/pdf.scss [hunk 2]
```diff
@@ -529,12 +528,17 @@ h6 {
     }
 
     thead td,
-    thead th {
-      background-color: $light;
+    thead th,
```
Comment (bspeare) [inline docroot/sites/admissions.uiowa.edu/modules/admissions_core/sass/pdf.scss:531]: We could probably move this back to Roboto.   Zilla is a smaller and heavier font and it is harder to read when it is rendered at a small size in all caps.

**Review by bspeare — CHANGES_REQUESTED** [review]

**Comment by bspeare** [comment]
PDF Updates look amazing!  Is this type of break expected?

Screen Shot 2021-05-28 at 9 08 29 AM

**Comment by dnkemper** [comment]
> PDF Updates look amazing! Is this type of break expected?
> 
> Screen Shot 2021-05-28 at 9 08 29 AM

CRAP! No! I will fix that! Need to add all potential use cases - so glad you found that!

**Review by bspeare — APPROVED** [review]
Styles and page breaks look good!

**Comment by dnkemper** [comment]
Yay! Thank you, @bspeare 🥳

**Pull request merged**